pkg/catalogv2/system/system_test.go

```diff
@@ -378,3 +391,212 @@ func TestIsInstalledExactVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestInstall(t *testing.T) {
+	t.Parallel()
+	asserts := assert.New(t)
+	type testInput struct {
+		namespace            string
+		name                 string
+		minVersion           string
+		exactVersion         string
+		values               map[string]interface{}
+		forceAdopt           bool
+		installImageOverride string
+	}
+	type testMocks struct {
+		indexOutput               *repo.IndexFile
+		indexError                error
+		isInstalledReleasesOutput []*release.Release
+		isInstalledReleasesError  error
+		hasStatusOutput           []*release.Release
+		hasStatusError            error
+		upgradeOutput             *catalog.Operation
+		upgradeError              error
+		podGetOutput              *v1.Pod
+		podGetError               error
+	}
+	type testCase struct {
+		name     string
+		input    testInput
+		mocks    testMocks
+		expected error
+		skip     bool
```

Comment:
Where is skip being used ?

Reply:
It's being used [here](https://github.com/rancher/rancher/pull/41931/files#diff-91741f593d7073b0f3166597105203f8396fcc31f6ffb4a6e8b2048daa176cbeR574), but all tests case are using the default false value. I added it in case we want to disable one test without deleting it, but I can remove it since it's not going to make a difference right now

Reply:
Let it be there. Its okay